source/examples/functions/MceStartStop/source.iecst

```diff
@@ -141,7 +143,7 @@ CASE io.nSmStartStop OF
   // -------------------------------------
   40:
     // Restart initiated by start button
-    IF bOsrStart THEN
+    IF bOsrHoldRestart THEN
```

Comment:
Perhaps we should check here for `mlx.NumberOfQueuedErrors= 0`, because if it ended up in this state by a robot alarm, we cannot restart before that alarm has been reset.